### PR TITLE
User update

### PR DESCRIPTION
**Pull request by zachholcomb**
# PR Description
Please include a summary of the change:
Adds default value to user image
Adds test for default value

Commit Emojis:
:100: Passing all tests

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: alembic/env.py [hunk 1]
```diff
@@ -57,8 +58,10 @@ def run_migrations_online():
     and associate a connection with the context.
 
     """
+    configuration = config.get_section(config.config_ini_section)
```
Comment (zachholcomb) [inline alembic/env.py:61]: This is for being able to run alembic migrations on heroku

File: app/schemas/user.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Config:
 class UserCreate(UserBase):
     email: EmailStr
     password: str
-    image: Optional[str]
+    image: Optional[str] = ""
```
Comment (zachholcomb) [inline app/schemas/user.py:20]: This adds default value to user image for FE

File: tests/models/test_user.py [hunk 3]
```diff
@@ -36,4 +36,11 @@ def test_create_user(db, cleanup):
     assert user.username == username
     assert user.image == image
 
+def test_user_image_has_default_value(db, cleanup):
+    email = "dan@example.com"
+    password = "123456"
+    username = "chunky_lover"
+    user_in = UserCreate(email=email, password=password, username=username)
+    user = crud.create_user(db, user=user_in)
+    assert user.image == ''
```
Comment (zachholcomb) [inline tests/models/test_user.py:45]: Test is passing!

Comment (QuinnrElder) [reply tests/models/test_user.py:45]: Great work Zack, thank you. Now the user profile can get rolling lol

**Pull request merged**